src/Shop.java

```diff
@@ -0,0 +1,167 @@
+import java.io.*;
+import java.util.ArrayList;
+
+ class Shop {
+    private ArrayList<Bike> bikes = new ArrayList<Bike>();
```

Comment:
Move this list to a `BikesDao` class

Reply:
Add next methods to a `BikesDao` class:
`Bike findOne(int id);`
`void update(int id, Bike  bike);`
`int save(Bike  bike);`
`void delete(int id)`

Reply:
ok